packages/core/src/browser/tree/tree-decorator.ts

```diff
@@ -282,6 +282,19 @@ export namespace TreeDecoration {
         readonly color?: Color;
     }
 
+    export interface TailDecorationCommand extends BaseTailDecoration {
```

Comment:
@kittaakos Could you review decorator part please?

Reply:
👍 Sure.